Tasks/app-store-promote/task.json

```diff
@@ -14,7 +14,7 @@
     "version": {
         "Major": "1",
         "Minor": "178",
-        "Patch": "2"
+        "Patch": "4"
```

Comment:
It's not necessary to bump version of other tasks - since we didn't apply any changes for them

Reply:
Fixed it. Thanks!